src/styles/saas-list.less

```diff
@@ -56,21 +96,45 @@
           font-size: 32px;
         }
       }
+
       .card-title {
-        vertical-align: middle;
-        display: table-cell;
+        color: inherit;
         padding-left: 8px;
       }
     }
-    .card-footer {
-      display: table-cell;
-      vertical-align: middle;
-      height: 80px;
 
-      .card-description {
-        font-size: 14px;
+    @media (min-width: @screen-sm-min) { // for card-container
+      display: block;
+
+      .card-header {
+        display: block;
+        text-align: center;
+        margin-bottom: 10px;
+
+        .card-icon {
+          font-size: 108px;
+          width: 100%;
+          margin-bottom: 20px;
+
+          &.fa.fa-mobile {
```

Comment:
@spadgett @cshinn I can't imagine that these will be font icons.  My guess is that it will be some custom icons or product imagery, but I will check on our side.  I think we will document that we want them to be monochrome though so that the color can still be changed?